addon/utils/game.py

```diff
@@ -39,5 +58,12 @@ def update_mapsrc(self, context):
 
 def verify(game):
     gameinfo = Path(game.game).joinpath('gameinfo.txt')
-    studiomdl = Path(game.bin).joinpath('studiomdl.exe')
-    return gameinfo.is_file() and studiomdl.is_file()
+    return gameinfo.is_file() and get_studiomdl_path(game).is_file()
+
+def get_studiomdl_path(game):
+    if os.name == 'posix' and (game.bin.endswith('linux32') or game.bin.endswith('linux64')):
+        p = Path(game.bin).joinpath('studiomdl')
+        if p.is_file():
+            return p
+    return Path(game.bin).joinpath('studiomdl.exe')
+
```

Comment:
Same point about variable names here.
Sorry to have so many nitpicks, I really do appreciate your contribution.

Reply:
This message is supposed to show up below the next one.
The aformentioned point about variable names is that I'd like longer ones.